workshops/dodge/README.md

```diff
@@ -64,50 +95,49 @@ Now we'll add something for the `draw()` function to do as well. Let's start by
 
 ```
 function draw() {
-  background(0,0,100);
+  background(0, 0, 100);
 }
 ```
 
-`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use background in some more ways, though, which are detailed in documentation [here](https://p5js.org/reference/#/p5/background).
+`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use `background` in some more ways, though, which are detailed in documentation [here](https://p5.js.org/reference/#/p5/background).
```

Comment:
Oh no, good catch. Let me fix that.

On Wed, Apr 6, 2016 at 9:18 AM, JESSICA <notifications@github.com> wrote:

> In workshops/dodge/README.md
> <https://github.com/jkwok91/hackclub/pull/2#discussion_r58735764>:
>
> >  }
> >  ```
> >
> > -`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use background in some more ways, though, which are detailed in documentation [here](https://p5js.org/reference/#/p5/background).
> > +`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use `background` in some more ways, though, which are detailed in documentation [here](https://p5.js.org/reference/#/p5/background).
>
> NO why did you not c your replace command
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jkwok91/hackclub/pull/2/files/f24ad8246cdf7e3e53e56b84653a167ecb9fe63a#r58735764>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>